.github/ISSUE_TEMPLATE/backport_release.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Backport Release
+about: Checklist for Backport releases
+
+---
+
+<!--
+Please check items along as you follow the release process.
+-->
+
+#### Overview
+
+This is a checklist for backport releases. This is filled in by both the releaser and the reviewer where necessary.
+
+#### Preparation
+
+- [ ] Create a release/<version> branch off the backport/<minor> branch
+```bash
+$ git checkout backport/<minor>
+$ git checkout -b release/${version}
+```
+- [ ] Cherry pick the necessary commits.
+```bash
+$ git cherrypick <commit>
+```
+- [ ] Update the `CHANGELOG.md` file as explained in the [Changelog Update](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/DEVELOPMENT.md#changelog-update) section.
+Once complete, you can add the file to staging
+```bash
+$ git add CHANGELOG.md
+```
+- [ ] Bump version as explained in the section [Version Bump](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/DEVELOPMENT.md#version-bump).
```

Comment:
If we're going to make things explicit, just include version bump steps here too. It's a backport, so we already know it's a bumpPatch.